x-pack/plugins/security_solution/public/common/components/header_page/index.tsx

```diff
@@ -43,7 +43,13 @@ const Header = styled.header.attrs({
 Header.displayName = 'Header';
 
 const FlexItem = styled(EuiFlexItem)`
-  display: block;
+  ${({ theme }) => css`
+    display: block;
+
+    @media only screen and (min-width: ${theme.eui.euiBreakpoints.m}) {
```

Comment:
I'm not sure if we should modify the width using media queries. Is this needed for some specific reason?

Reply:
the existing behavior is when the width drops below m, the title and the supplements become full width and take 2 lines. without the media query, the title stays 50% width leaving a bunch of awkward empty space on the right when under m.